### PR TITLE
support xxx/yyy/global

### PR DESCRIPTION
**Pull request by dorsradek**
- Support xxx/yyy/global/zzz:
  - application/hydra/global/s3
  - infrastructure/launch-darkly/global/license
- Sort application.properties and uniq

### CODE REVIEW

**Review by maximelaboisson — APPROVED** [review]
LGTM, let's test with rc chart once merged though 🙌

**Comment by dorsradek** [comment]
@maximelaboisson I am stupid. We don't have `global` in the path anymore. I will create another PR

**Comment by maximelaboisson** [comment]
> @maximelaboisson I am stupid. We don't have `global` in the path anymore. I will create another PR

My brain is fried, I didn't realise either

**Pull request merged**
